docs/support/migration.md

```diff
@@ -0,0 +1,207 @@
+---
+title: Migration
+---
+
+# Migrating between SDK versions
+
+While the Packs team strives to maintain backwards compatibility between SDK versions, there may occasionally be times where you need to change your code to work with a new SDK version. This page will include instructions on how to perform the migration. A detailed list of all changes in the SDK can be found in the [Changelog][changelog].
+
+
+## Version 0.11.0
```

Comment:
Probably fine but may be worth extra emphasis that they must have upgraded to version 0.11.0 in order to be able to attempt migration. (And we should provide the command for upgrading if they use the CLI.)

Reply:
Done. Added some new sections to the permanent Migration guide, and moved the version-specific stuff to a sub-page.